app/controllers/chat_channel_memberships_controller.rb

```diff
@@ -117,7 +117,9 @@ def respond_to_invitation(previous_status)
         flash[:settings_notice] = "Invitation to  #{channel_name} accepted. It may take a moment to show up in your list."
       else
         send_chat_action_message("@#{current_user.username} added @#{@chat_channel_membership.user.username}", current_user, @chat_channel_membership.chat_channel_id, "joined")
-        NotifyMailer.channel_invite_email(@chat_channel_membership, @chat_channel_membership.user).deliver_later
+
+        NotifyMailer.with(membership: @chat_channel_membership, inviter: @chat_channel_membership.user).channel_invite_email.deliver_later
```

Comment:
this is the new API in Rails 6 for parameterized mailers: https://github.com/rails/rails/pull/34591

Reply:
Oh this is super neat!

Reply:
Whoa! Gotta teach myself to start thinking in the Rails 6 way 😅